### PR TITLE
Spam generator tool

### PR DESCRIPTION
**Pull request by tharindal**
Signed-off-by: Tharinda Lunuwila <tharinda.gl@gmail.com>

### CODE REVIEW

File: saito-spammer/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/target
```
Comment (SankaD) [inline saito-spammer/.gitignore:1]: /target or target/ ?

**Review by SankaD — APPROVED** [review]
approving to merge. not reviewing. can be tested with node testing

File: saito-core/src/core/data/configuration.rs [hunk 2]
```diff
@@ -23,10 +23,18 @@ pub struct Server {
...
+    pub timer_in_milli: u64,
+    pub burst_count: u32,
+    pub bytes_per_tx: u32,
+}
+
 #[derive(Deserialize, Debug)]
 pub struct Configuration {
     pub server: Server,
     pub peers: Vec<PeerConfig>,
+    pub spammer: Spammer,
```
Comment (SankaD) [inline saito-core/src/core/data/configuration.rs:37]: this shouldn't be here

File: saito-rust/configs/saito.config.template.json [hunk 3]
```diff
@@ -16,5 +16,10 @@
       "protocol": "http",
       "synctype": "full"
     }
-  ]
+  ],
+  "spammer": {
```
Comment (SankaD) [inline saito-rust/configs/saito.config.template.json:20]: this should be removed

Comment (tharindal) [reply saito-rust/configs/saito.config.template.json:20]: Okay will remove this in a future commit, adding as a TODO

**Pull request merged**